templates/components/engines/typescript/agent/tools/index.ts

```diff
@@ -35,6 +36,9 @@ const toolFactory: Record<string, ToolCreator> = {
     );
     return await openAPIActionTool.toToolFunctions();
   },
+  "duckduckgo.DuckDuckGoSearchToolSpec": async (config: unknown) => {
```

Comment:
In python, it's a llamahub tool but in TS it's a local tool. Do you have any suggestions?

Reply:
just a comment would be good